### ISSUE TITLE
Implement OpenSRP OAuth username change

### ISSUE THREAD

**Issue by moshthepitt** [onaio/superset-patchup#50]
v2 of OpenSRP has changed the username field from `userName` to `username`.  We need to make this change [here](https://github.com/onaio/superset-patchup/blob/master/superset_patchup/oauth.py#L291), in a backwards compatible way.

**Comment by moshthepitt** [comment]
```sh
 ~/superset-patchup git tag
v0.1.0
v0.1.1
v0.1.2
v0.1.3
v0.1.4
v0.1.5
v0.1.6
```

This change is immediately needed for `Reveal Stage` and `Reveal Zambia` which are both on Superset `0.29.0rc7`.  My worry is I am not sure if the latest version of `Ketchup` is compatible with that version of Superset.

@ukanga I see that for [Hello Mwana](https://github.com/onaio/playbooks/pull/2502/files#diff-d3f076d2b51f4ca1f8d30189796b8d78R119) (which is essentially the same version of Superset) you used a branch `fix-redirect-v0.1.6` - should we tag a release off of this branch?

**Comment by ukanga** [comment]
@moshthepitt Yes, you can tag a release from that branch.

**Comment by ukanga** [comment]
As to the access of the username
```python
...
username = user_object.get("username") or user_object.get('userName').
...
```

**Comment by moshthepitt** [comment]
Fixed by #49 and #51
